jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -150,11 +152,17 @@ protected void navigateToPerspective(final String perspectiveId,
                              parameterValue);
         placeManager.goTo(request);
     }
-
+    
     public boolean isUserAuthorizedForPerspective(final String perspectiveId) {
         final ResourceRef resourceRef = new ResourceRef(perspectiveId,
                                                         ActivityResourceType.PERSPECTIVE);
         return authorizationManager.authorize(resourceRef,
                                               identity);
     }
+    
+    public void openErrorView(final String parameterId) {}
+    
+    public Predicate<T> getViewErrorsActionCondition(){
+        return null;
```

Comment:
@mcivantos-tribalyte I would avoid returning ```null``` and make this method abstract instead, but it's your choice.

Reply:
@bsremac At first I made this method abstract, but Cristiano suggested change it in order to overriding only where needed.